header.go

```diff
@@ -2028,6 +2478,10 @@ func (h *ResponseHeader) parseHeaders(buf []byte) (int, error) {
 					}
 					continue
 				}
+				if caseInsensitiveCompare(s.key, strTrailer) {
```

Comment:
```suggestion
				} else if caseInsensitiveCompare(s.key, strTrailer) {
```

Reply:
The previous judgement has a `continue`, would it be clearer not to write `else` ?